packages/web/src/components/back-button/BackButton.module.css

```diff
@@ -1,39 +1,36 @@
 .backButton {
   height: 24px;
   width: 24px;
-  background-color: var(--neutral-light-5);
-
+  padding: 0;
+  margin-right: var(--unit-4);
   display: flex;
   align-items: center;
   justify-content: center;
-
-  border-radius: 50%;
-
-  transform: rotate(180deg);
-  transition: all 0.07s ease-in-out;
-  cursor: pointer;
-  user-select: none;
+  transition: 0.12s ease-out !important;
 }
-.backButton.light {
-  background-color: var(--neutral-light-6);
+
+.backButton:hover {
+  transform: scale3d(1.11, 1.11, 1.11) !important;
```

Comment:
not ideal use of important, but will need a new harmony button of some kind to handle this case

Reply:
nice. Maybe put a todo here to update it once we've made the harmony component? or a linear item with reference to this file